app/javascript/components/editor/TestsGroupedByStatusList.tsx

```diff
@@ -2,8 +2,45 @@ import React from 'react'
 import { TestStatus, Test } from './types'
 import { GraphicalIcon } from '../common/GraphicalIcon'
 import { TestsGroup, TestWithToggle } from './TestsGroup'
+import pluralize from 'pluralize'
+import { TestSummary } from './TestSummary'
 
-export function TestsGroupList({
+const Tests = ({
+  tests,
+  language,
+}: {
+  tests: TestWithToggle[]
+  language: string
+}): JSX.Element => {
+  return (
+    <>
+      {tests.map((test) => (
+        <TestSummary
+          key={test.name}
```

Comment:
Probably want to check somewhere that all test names are unique, even though they should!